test/features/calendar.feature

```diff
@@ -1,217 +1,206 @@
+            | name        | "Livestream XY"                       |
+            | description | "descriptive Description"             |
+            | url         | "https://domain.com"                  |
+            | platform    | "YouTube"                             |
+            | author      | {"platform":"discord","uid":"hubber"} |
+            | startDate   | "2021-01-01T00:00:00.000Z"            |
+            | endDate     | "2021-01-01T00:00:00.000Z"            |
+        And the response should contain:
+            | documentId | "TYPE:ID" |
+        When make a PUT request to "/calendar/321" with:
+            | name        | "Livestream YZ"                      |
+            | description | "undescriptive Description"          |
+            | url         | "https://mydomain.com"               |
+            | platform    | "Twitch"                             |
+            | author      | {"platform":"discord","uid":"hubby"} |
+            | startDate   | "2021-01-01T00:00:00.000Z"           |
+            | endDate     | "2021-01-01T00:00:00.000Z"           |
+        Then the response status code should be 404
+        And the response should contain:
+            | message    | "no event for 321 found" |
+            | statusCode | 404                      |
 
-#     Scenario: delete non-existing event
-#         Given authorisation
-#         And make a POST request to "/calendar" with:
-#             | name        | "Livestream XY"                       |
-#             | description | "descriptive Description"             |
-#             | url         | "https://domain.com"                  |
-#             | platform    | "YouTube"                             |
-#             | author      | {"platform":"discord","uid":"hubber"} |
-#             | startDate   | "2021-01-01T00:00:00.000Z"            |
-#             | endDate     | "2021-01-01T00:00:00.000Z"            |
-#         When make a DELETE request to "/calendar/321"
-#         Then the response status code should be 404
-#         And the response should contain:
-#             | statusCode | 404               |
-#             | message    | "Event Not Found" |
+    Scenario: delete an event
+        Given authorisation
+        When make a POST request to "/calendar" with:
+            | name        | "Livestream XY"                       |
+            | description | "descriptive Description"             |
+            | url         | "https://domain.com"                  |
+            | platform    | "YouTube"                             |
+            | author      | {"platform":"discord","uid":"hubber"} |
+            | startDate   | "2021-01-01T00:00:00.000Z"            |
+            | endDate     | "2021-01-01T00:00:00.000Z"            |
+        And the response should contain:
+            | documentId | "TYPE:ID" |
+        When make a DELETE request to "/calendar/{id}"
+        Then the response status code should be 204
```

Comment:
I think we should protect the details to be the author of the resource, what do you think? this can come in another issue + PR

Reply:
Yes thats true. Thats actually something I didnt really think about so far

Reply:
We would probably have to handle this with a query-parameter if this is allowed
>
>A payload within a DELETE request message has no defined semantics;
   sending a payload body on a DELETE request might cause some existing
   implementations to reject the request.
>
source: [here](https://datatracker.ietf.org/doc/html/rfc7231#page-29)

Reply:
Good idea 👍 

Or maybe we only allow `admin` users to delete as that might be easier. We should probably raise an issue and add this discussion there

Reply:
We can continue the discussion here https://github.com/EddieHubCommunity/api/issues/107